src/genie/libs/parser/iosxe/show_ospf.py

```diff
@@ -4331,26 +4344,36 @@ def cli(self, neighbor='', output=None):
                 router_id = None
                 bfd_state = m.groupdict().get('bfd_state', None)
                 # Get OSPF process ID from 'show ip ospf interface'
-                cmd = 'show ip ospf interface {}'.format(interface)
-                out = self.device.execute(cmd)
+                flag = False
 
-                for line in out.splitlines():
+                for line in ospfint_out.splitlines():
                     line = line.rstrip()
 
                     # Process ID 2, Router ID 10.229.11.11, Network Type SHAM_LINK, Cost: 111
-                    p = re.search('Process +ID +(?P<instance>(\S+)), +Router +ID'
-                                  ' +(?P<router_id>(\S+)) +(.*)', line)
+                    # p = re.search('Process +ID +(?P<instance>(\S+)), +Router +ID'
+                    #                 ' +(?P<router_id>(\S+)) +(.*)', line)
```

Comment:
This can be removed if it's requires commenting

Reply:
Done! Thanks for letting me know.